README.md

```diff
@@ -7,58 +8,65 @@
 [![Slack](https://ilios-slack.herokuapp.com/badge.svg)](https://ilios-slack.herokuapp.com/)
 [![Stories in Ready](https://badge.waffle.io/ilios/frontend.png?label=ready&title=Ready)](https://waffle.io/ilios/frontend)
 
-## Web interface for accessing and managing [Ilios Platform](https://github.com/ilios/ilios) data.
+## Open Source Heroes
+
+Every day these for profit companies make developing Ilios possible.  Without the free tier of service they offer to
+open source software we would be nowhere so if you have a chance to pay them for something please do.
+
+- [GitHub](https://github.com)
+- [Travis CI](https://travis-ci.org/)  
+- [Sauce Labs](https://saucelabs.com/)
+- [Heroku](https://www.heroku.com)
+- [Netlify](https://www.netlify.com)
+- [BrowserStack](https://www.browserstack.com)
 
-### Directions for developers working with this ember-cli app
+## Directions for developers working with this ember-cli app
 
-#### Prerequisites
+### Prerequisites
 
 You will need the following things properly installed on your computer.
 
 * [Git](https://git-scm.com/)
-* [Node.js](https://nodejs.org/)
+* [Yarn](https://yarnpkg.com/)
+* [Ember CLI](https://ember-cli.com/)
+* [Google Chrome](https://google.com/chrome/)
+
+### Installation
 
-We recommend installing Node.js though [NVM](https://github.com/creationix/nvm#installation)
+* `git clone git@github.com:ilios/frontend.git` this repository
+* `cd ilios-frontend`
+* `yarn install`
 
-#### Installation
+### Running / Development
 
```

Comment:
Once it is up and running, the Frontend will need to know where your backend API server is located.  You can set this information in a system wide environment variable name `ILIOS_FRONTEND_API_HOST` or can just add the setting to a file named `.env` file within the same directory as your Ilios Frontend code:

`echo "ILIOS_FRONTEND_API_HOST=https://ilios3-demo.ucsf.edu" > .env`